.yaydoc.yml

```diff
@@ -0,0 +1,35 @@
+metadata:
+  author: FOSSASIA
+  projectname: "Susi Server"
+  version: development
+build:
+  theme:
+    name: sphinx_fossasia_theme   
+  source: docs
+  logo: images/susi-logo-original.svg
+  subproject:
+    - url: https://github.com/fossasia/susi_android
+      source: docs
```

Comment:
can I keep it as it or should I 
.


























































































































































































































..................
.






















































































































































































































































































































































































































@imujjwal96 can I keep as old or go with made changes